controllers/csiaddons/reclaimspacejob_controller.go

```diff
@@ -403,7 +403,7 @@ func (r *ReclaimSpaceJobReconciler) nodeReclaimSpace(
 		target.nodeID,
 		identity.Capability_ReclaimSpace_ONLINE)
 	if nodeClient == nil {
-		return nil, errors.New("node Client not found")
+		return nil, fmt.Errorf("node Client not found for %s nodeID", target.nodeID)
```

Comment:
```suggestion
		return nil, fmt.Errorf("node Client not found for %q nodeID", target.nodeID)
```
add quotes so it stands out ?